frontend/src/components/ViewActivities/editActivityFormElements.js

```diff
@@ -10,13 +10,13 @@ const TIMEWIDTH = 2;
 const TZPICKERWIDTH = 3;
 
 /**
- * Create a Text element Form Group for the editActivity form.
- *
- * @param {string} controlId FormGroup's control ID.
- * @param {string} formLabel The label of the field for this FormGroup.
- * @param {string} placeHolder The input's placeholder.
- * @param {ref} ref The input's reference.
- * @returns {HTML} A text element form group.
+ * Create a Text element Form Group for the editActivity form. 
+ * 
+ * @param {!string} controlId FormGroup's control ID.
+ * @param {!string} formLabel The label of the field for this FormGroup. 
+ * @param {!string} placeHolder The input's placeholder. 
+ * @param {?ref} ref The input's reference.
```

Comment:
```suggestion
 * @param {?React.RefObject} ref The input's reference.
```

Reply:
Last time I will do this one too. Keep an eye out for other instances if you want to make the change.